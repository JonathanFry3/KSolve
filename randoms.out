Unknown.
Took 20.4 sec. 10 million unique states.
Unknown.
Took 28.26 sec. 10 million unique states.
Minimal solution in 108 moves + 21 flips.
Took 21.02 sec. 8.099 million unique states.
Minimal solution in 112 moves + 21 flips.
Took 13.06 sec. 5.352 million unique states.
Minimal solution in 104 moves + 21 flips.
Took 0.483 sec. 0.07109 million unique states.
Minimal solution in 110 moves + 21 flips.
Took 7.96 sec. 3.537 million unique states.
Unknown.
Took 28.76 sec. 10 million unique states.
Minimal solution in 122 moves + 21 flips.
Took 2.864 sec. 1.082 million unique states.
Minimal solution in 115 moves + 21 flips.
Took 7.936 sec. 3.215 million unique states.
Impossible.
Took 2.468 sec. 0.4419 million unique states.
