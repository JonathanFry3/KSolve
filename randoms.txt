# Random test deals
game: 49
game: 50
game: 51
game: 52
game: 53
game: 54
game: 55
game: 56
game: 57
game: 58
