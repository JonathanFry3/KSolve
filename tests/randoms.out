Minimal solution in 118 moves + 21 flips in 2 passes.
Took 16.12 sec. 37.55 million moves generated.
Minimal solution in 131 moves + 21 flips in 3 passes.
Took 13.68 sec. 23.21 million moves generated.
Minimal solution in 108 moves + 21 flips in 2 passes.
Took 3.301 sec. 8.286 million moves generated.
Minimal solution in 112 moves + 21 flips in 2 passes.
Took 1.846 sec. 5.562 million moves generated.
Minimal solution in 104 moves + 21 flips in 2 passes.
Took 0.04779 sec. 0.0691 million moves generated.
Minimal solution in 110 moves + 21 flips in 2 passes.
Took 1.087 sec. 3.688 million moves generated.
Minimal solution in 110 moves + 21 flips in 2 passes.
Took 10.38 sec. 22.39 million moves generated.
Minimal solution in 122 moves + 21 flips in 3 passes.
Took 0.4594 sec. 1.155 million moves generated.
Minimal solution in 115 moves + 21 flips in 2 passes.
Took 1.185 sec. 3.462 million moves generated.
Impossible.
Took 0.3261 sec. 0.4561 million moves generated.
